src/categorical_algebra/HomSearch.jl

```diff
@@ -115,10 +116,29 @@ homomorphisms exist, it is exactly as expensive.
 homomorphisms(X::ACSet, Y::ACSet; alg=BacktrackingSearch(), kw...) =
   homomorphisms(X, Y, alg; kw...)
 
-function homomorphisms(X::ACSet, Y::ACSet, alg::BacktrackingSearch; kw...) 
+"""
```

Comment:
Thanks for documenting this method! Can we add a line the beginning, before documenting the optional arguments, stating what the method does?